TUTORIAL.md

```diff
@@ -0,0 +1,384 @@
+# ZNAP Tutorial - **Donate/Pay a user**
+
+## Description
+
+Alice is a well-known influencer on Twitter. She usually shares research and valuable insights with her audience free of charge. She decides she wants to start a donation campaign for herself. She spins up a “Donate to me” Blink using the Actions Stack and shares the link to Sphere on Twitter.The link unfurls into an actionable Blink containing the following information:
+
+1. Her image, name, description
+2. 3 buttons: 1 SOL, 5 SOL, 10 SOL
+3. 4th button with an input field: “Enter a custom SOL amount”
+
+## Brief review
+
+Solana Actions are APIs that adhere to the Solana Actions Specification. Our Solana Actions must be capable of returning:
+
+1. Action Metadata
+2. A transaction to be sent to the Solana blockchain.
+
+## Let's Start!
+
+First, we will create our workspace using `znap-cli`. If you do not have `znap-cli` installed, follow our guide [here](https://github.com/heavy-duty/znap/blob/master/INSTALLATION.md).
+
+Next, let's create a new project with `znap init <project-name>`.
```

Comment:
I'd like to include some information about what happens when `init` is called, which files are created, where, etc... similar to the case of `new` in step 1.

Reply:
Done